### ISSUE TITLE
Add full SuperTable support

### ISSUE THREAD

**Issue by Emkaytoo** [Pennebaker/craft-architect#10]
The Architect exported a majority of a SuperTable field, but didn't export the fields within the table itself.

Architect output is below.

```
{
  "fieldGroups": [
    "Assets"
  ],
  "fields": [
    {
      "group": "Assets",
      "name": "Location Contacts",
      "handle": "locationContacts",
      "instructions": "Add rows below to generate the contact table. Add a label for the section, and then select the appropriate contacts from the list.",
      "type": "verbb\\supertable\\fields\\SuperTableField",
      "minRows": "1",
      "maxRows": "",
      "localizeBlocks": false,
      "staticField": "",
      "fieldLayout": "row",
      "selectionLabel": "Add contact"
    }
  ]
}
```

**Comment by Emkaytoo** [comment]
Did you get SuperTable working within a Matrix? I'm not able to import a SuperTable within a matrix *using the json exported by the Architect after configuring it within the control panel*.

**Comment by spAnser** [comment]
This is actually an issue with how the Matrix fields are parsed.

**Comment by spAnser** [comment]
Fixed in 2.2.3
